NEWS.md

```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```

Comment:
Opinions on whether to actually link the PRs? I kind of wish GitHub would automatically do so.